README.md

```diff
@@ -0,0 +1,32 @@
+![gif](https://media.giphy.com/media/MAzusM3FqQzcl4crXq/giphy.gif)
+
+# Echo Server Requirements
+- Echo Server connects to a client
+- Once running, anything typed in the client will be sent to Server
+- Server will echo back the message to the client
+
+# Local Development Setup
+### 1. Install Java
+Check if Java is present: `java -version`
```

Comment:
I noticed that you checked in the `gradle` executable and jar file which is frowned upon. This is considered a security issue since someone can alter them to run malicious code. The best practice is to leave it out of your repository, and document `gradle` as a dependency that users can install onto their own computers.